src/schema/v2/fair_organizer.ts

```diff
@@ -23,6 +27,69 @@ export const FairOrganizerType = new GraphQLObjectType<any, ResolverContext>({
       about: {
         type: GraphQLString,
       },
+      articlesConnection: {
+        description: "A connection of articles related to a partner.",
+        type: articleConnection.connectionType,
+        args: pageable({
+          sort: ArticleSorts,
+          page: { type: GraphQLInt },
+          inEditorialFeed: {
+            type: GraphQLBoolean,
+            description:
+              "Articles that are ready to be publicly viewed in the feed by everyone.",
+          },
+        }),
+        resolve: async (
+          { profile_id },
+          args: {
+            inEditorialFeed?: Boolean
+            sort?: ArticleSort
+          } & CursorPageable,
+          { articlesLoader, fairsLoader }
+        ) => {
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          const gravityOptions = {
+            fair_organizer_id: profile_id,
+            size: 100,
+          }
+
+          const { body } = await fairsLoader(gravityOptions)
```

Comment:
Note that when both `articlesConnection` and `fairsConnection` are queried you'll see two requests to the underlying `/api/v1/fairs?fair_organizer_id=[slug]` API endpoint in Gravity.